File: 0205-isomorphic-strings/0205-isomorphic-strings.cpp
class Solution {
public:
    bool isIsomorphic(string s, string t) {
        unordered_map<char,char> m1,m2;
        if(s.length()!=t.length()) return false;
        for(int i=0;i<s.length();i++){
            if(m1[s[i]] && m1[s[i]]!=t[i]) return false;
            if(m2[t[i]] && m2[t[i]]!=s[i]) return false;
            m1[s[i]]=t[i];
            m2[t[i]]=s[i];
        }
        return true;

        
    }
};

File: 0205-isomorphic-strings/0205-isomorphic-strings.py
class Solution(object):
    def isIsomorphic(self, s, t):
        map1 = []
        map2 = []
        for i in s:
            map1.append(s.index(i))
        for j in t:
            map2.append(t.index(j))
        if map1==map2:
            return True
        return False
        """
        :type s: str
        :type t: str
        :rtype: bool
        """

        